src/frontend/state/GlobalState.tsx

```diff
@@ -673,11 +637,8 @@ class GlobalState extends PureComponent<Props> {
 
   async componentDidMount() {
     const { t } = this.props
-    const { epic, gameUpdates = [], libraryStatus, category } = this.state
-    const oldCategory: string = category
-    if (oldCategory === 'epic') {
-      this.handleCategory('all')
-    }
```

Comment:
more than a year ago we migrated the `epic` filter to be called `legendary`, by now any user should already have the correct store filter saved in the local storage so we can remove this migration workaround